eksconfig/spec.go

```diff
@@ -0,0 +1,7 @@
+package eksconfig
+
+// Spec contains specs for addons
+type Spec struct {
+	// ClusterAutoscaler defines config for the cluster autoscaler
+	ClusterAutoscaler *ClusterAutoscalerSpec `json:"clusterAutoscaler,omitempty"`
```

Comment:
`clusterAutoscaler` seems consistent with upstream Kubernetes, but makes it inconsistent with current `eksconfig` JSON field names used for env parsing.

Reply:
Yeah the goal here was to align with upstream kubernetes whereever possible. I also wasn't intending for these addons to be configurable via ENV vars and to only use env vars for the older addons. What do you think?

Reply:
Also, https://github.com/iancoleman/strcase can easily convert camelCase or kabob-case into UPPER_SNAKE_CASE

Reply:
Good point. We can discuss this maybe in next iteration for full compatibilities.